src/main/java/com/puppycrawl/tools/checkstyle/checks/design/ThrowsCountCheck.java

```diff
@@ -70,13 +104,72 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public void failTest() throws IllegalStateException,
+ *                                 ArrayIndexOutOfBoundsException,
+ *                                 NullPointerException { // violation, max allowed is 2
+ *         // body
+ *     }
+ *
+ *     public void successTest() throws ArithmeticException,
+ *                                 NumberFormatException { // ok
+ *         //body
+ *     }
+ *
+ *     private void privateTest() throws CloneNotSupportedException,
+ *                                 ClassNotFoundException,
+ *                                 IllegalAccessException,
+ *                                 ArithmeticException,
+ *                                 ClassCastException { // ok
+ *         // body
+ *     }
+ *
+ * }
+ * </pre>
+ * <p>
  * To configure the check so that it doesn't skip private methods:
  * </p>
  * <pre>
  * &lt;module name="ThrowsCount"&gt;
  *   &lt;property name=&quot;ignorePrivateMethods&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public void failTest() throws CloneNotSupportedException,
+ *                                 ArrayIndexOutOfBoundsException,
+ *                                 StringIndexOutOfBoundsException,
+ *                                 IllegalStateException,
+ *                                 NullPointerException { // violation, max allowed is 4
+ *         // body
+ *     }
+ *
+ *     public void successTest() throws ArithmeticException,
+ *                                 NumberFormatException { // ok
+ *         //body
+ *     }
+ *
+ *     private void privateTest() throws CloneNotSupportedException,
+ *                                 ClassNotFoundException,
+ *                                 IllegalAccessException,
+ *                                 ArithmeticException,
+ *                                 ClassCastException { // violation, max allowed is 4
+ *         // body
+ *     }
+ *
+ *     private void successTest() throws IllegalStateException,
+ *                                 NullPointerException { // ok
+ *         //body
```

Comment:
can add a space, `// body`

Reply:
Done @esilkensen

Reply:
thanks - looks like this is the last one